ship.config.js

```diff
@@ -14,6 +14,12 @@ module.exports = {
     exec('yarn doctoc');
   },
   pullRequestTeamReviewer: ['instantsearch-for-websites'],
+  afterPublish: ({ exec }) => {
```

Comment:
Do we want to release on this tag only for stable versions?

Reply:
Good call, we don't want a prerelease, or a release of a previous major to override this

Reply:
It makes sense. Now it's releasing types only with v4 stable version.
https://github.com/algolia/instantsearch.js/pull/4307/commits/1e51fea844e2ae6f074efe82a574b1916ec00ea1